src/main/kotlin/de/bund/digitalservice/ris/norms/domain/entity/Norm.kt

```diff
@@ -1,5 +1,5 @@
 package de.bund.digitalservice.ris.norms.domain.entity
 
-import de.bund.digitalservice.ris.norms.domain.value.Guid
+import java.util.UUID
 
-data class Norm(val guid: Guid, val longTitle: String, val articles: List<Article> = listOf())
+data class Norm(val guid: UUID?, val longTitle: String, val articles: List<Article> = listOf())
```

Comment:
Why are all (`Norm`, `Article` and `Paragraph`) `guid` properties now optional (or rather null-able)? I guess that is a leftover? 🤔 
I mean from the perspective of the Domain, there is never an instance of such an entity without a `guid`. So it should not be nullable, because the whole domain and everyone working with it should not need to handle this null case at all. Because it can't be.

Reply:
The nullable feature of id in specific at domain level is mandatory since we will generate it in the persistence layer and that it should be a unique primary key for sorting in the database. If we are going to use another auto-generated key for indexing I would have agreed with you for not making it nullable. This is the best practice in kotlin especially when you separate domain entities from dtos. You can take a look at this: https://blog.codecentric.de/crud-operations-spring-rest-resources-kotlin